### PR TITLE
Explain that BIP148 may not lead to the activation of SegWit.

### PR DESCRIPTION
**Pull request by da2ce7**
I found this quite non obvious; so I have made a pull request to fix the wording.

Yes, my last pull request was ill-conceived, thank-you as-always for the review.

### CODE REVIEW

**Comment by luke-jr** [comment]
@shaolinfry 

Suggest moving this outside Specification, and explaining that segwit can be re-proposed as a normal UASF in this circumstances.

File: bip-0148.mediawiki [hunk 1]
```diff
@@ -24,7 +24,9 @@ This document specifies a BIP16 like soft fork flag day activation of the segreg
 
 Segwit increases the blocksize, fixes transaction malleability, and makes scripting easier to upgrade as well as bringing many other [https://bitcoincore.org/en/2016/01/26/segwit-benefits/ benefits].
 
-It is hoped that miners will respond to this BIP by activating segwit early, before this BIP takes effect. Otherwise this BIP will cause the mandatory activation of the existing segwit deployment before the end of midnight November 15th 2017.
+It is hoped that miners will respond to this BIP by activating segwit early, before this BIP takes effect. Otherwise this BIP may cause the mandatory activation of the existing segwit deployment before the end of midnight November 15th 2017.
```
Comment (jonathancross) [inline bip-0148.mediawiki]: "_**before the end of** midnight_" can simply be written as "_**before** midnight_"  (I know you didn't write this part, but would be nice to update if making changes to this line)

File: bip-0148.mediawiki [hunk 2]
```diff
@@ -24,7 +24,9 @@ This document specifies a BIP16 like soft fork flag day activation of the segreg
...
+
+In the cases that the hashrate drops to below 14% over 3.5 months sustained, it is possible that SegWit is not locked-in before November 15th 2017.
```
Comment (jonathancross) [inline bip-0148.mediawiki]: If the hashrate drops below 14% for a sustained 3.5 month period, it is possible that segwit will not be locked-in before November 15th 2017.

**Comment by da2ce7** [comment]
@jonathancross, thank-you for your wording suggestions, I have adopted them in this pull request.

**Comment by murchandamus** [comment]
While the changes seem reasonable to me, it looks like the author has not shown interest in almost 7 years to add this amendment to their BIP. As such, I’m closing this PR now, but please feel free to ask to reopen it, if you would like to continue pursuing a merge.

**Pull request closed without merging**